system/Test/CIDatabaseTestCase.php

```diff
@@ -135,6 +142,8 @@ public function __construct()
 	 */
 	public function setUp()
 	{
+		$this->loadDependencies();
```

Comment:
If I don't setup correct database config for testing, I would get the error like the following:
~~~
$ ./phpunit --exclude-group DatabaseLive 

An uncaught Exception was encountered

Type:        ErrorException
Message:     mysqli::real_connect(): (HY000/1049): Unknown database 'xci4'
Filename:    .../CodeIgniter4/system/Database/MySQLi/Connection.php
Line Number: 181
~~~

But you are fine, so I will revert it.